### ISSUE TITLE
Swift Compiler Warnings

### ISSUE THREAD

**Issue by n3trino** [CardinalNow/iOS-CircleProgressView#55]
Hi,

I updated to CircleProgressView (1.1.2) and i still get compiler warnings here:

```
        let startAngle:CGFloat = clockwise ? CGFloat(-internalProgress * M_PI / 180.0) : CGFloat(constants.twoSeventyDegrees * M_PI / 180)
        let endAngle:CGFloat = clockwise ? CGFloat(constants.twoSeventyDegrees * M_PI / 180) : CGFloat(-internalProgress * M_PI / 180.0)
 
        progressPath.addArc(withCenter: center, radius:radius, startAngle:startAngle, endAngle:endAngle, clockwise:!clockwise)
        let r = radius - trackWidth * 0.5
        if roundedCap {
            let capCenter = CGPoint(x: center.x + r * cos(endAngle), y: center.y + r * sin(endAngle))
        
            progressPath.addArc(withCenter: capCenter, radius: trackWidth * 0.5, startAngle: endAngle, endAngle: endAngle + CGFloat(M_PI), clockwise: !clockwise)
        }
        progressPath.addArc(withCenter: center, radius:radius-trackWidth, startAngle:endAngle, endAngle:startAngle, clockwise:clockwise)
        
        if roundedCap {
        let capCenter = CGPoint(x: center.x + r * cos(startAngle), y: center.y + r * sin(startAngle))
        
            progressPath.addArc(withCenter: capCenter, radius: trackWidth * 0.5, startAngle: startAngle, endAngle: startAngle + CGFloat(M_PI), clockwise: clockwise)
        }
```

**Comment by BumwooPark** [comment]
i got compiler warning too (1.1.2)

**Comment by murad1981** [comment]
me too .. please replace the PI things with the new .pi

**Comment by jdrevet** [comment]
It's weird because the warnings have been fixed in 1.1.2. But Cocoapods seems to download an older version whereas the version 1.1.2 is mentioned in Podfile.lock. I don't understand ...

**Comment by xrolfex** [comment]
Will be fixed in next release.
